@here/harp-mapview-decoder/lib/GeoJsonTiler.ts

```diff
@@ -85,27 +85,7 @@ export class GeoJsonTiler implements ITiler {
         const tile = index.getTile(tileKey.level, tileKey.column, tileKey.row);
         if (tile !== null) {
             tile.layer = indexId;
-            for (const feature of tile.features) {
-                feature.originalGeometry = this.getOriginalGeometry(feature, index.geojson);
```

Comment:
Here the original geometry was attached to the output features, but it's never used. It was previously used for lines, but Roberto changed that code half a year ago, already discussed with him.